src/docs/arc42/adrs/ADR002-Technology-Stack.adoc

```diff
@@ -0,0 +1,91 @@
+= ADR-002: Technology Stack Selection - Python 3.8+ and MCP Protocol
+
+|===
+| Datum:    | 2025-01-13
+| Autoren:  | AsciiDoc MCP Development Team
+| Status:   | Accepted
+|===
+
+== Problem
+
+The AsciiDoc MCP Server requires a technology stack that balances developer productivity, ecosystem maturity, performance, and compatibility with the Model Context Protocol specification.
+
+== Alternatives Considered
+
+[options="header",cols="2,2,2,2"]
+|===
+|Technology|Pros|Cons|Score
+|**Python 3.8+**|• Mature AsciiDoc ecosystem +
```

Comment:
an AsciiDoc List within a table needs a) the "*" marken and b) the cell needs to me marked as AsciiDoc-Cell by starting it with "a|".
Please fix all tables in the architecture accordingly

Reply:
add these rules to the custom instructions file